### PR TITLE
Reveal info

### PR DESCRIPTION
**Pull request by ryanmansfield**

### CODE REVIEW

File: app/views/bookings/_arrival-confirmed.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="container" id="driver-confirmed">
```
Comment (bcorrell7) [inline app/views/bookings/_arrival-confirmed.html.erb]: change id to arrival-confirmed

File: app/views/bookings/_arrival-confirmed.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+   <div class="row ct-section_header ct-section_header--type2"></div>
+   <div class="row trip-details">
+      <div class="venue-details col-9">
+        <div class="container">
+          <div class="progress-bar">
+            <div class="loaded-driver-confirmed"></div>
```
Comment (bcorrell7) [inline app/views/bookings/_arrival-confirmed.html.erb]: change to loaded-arrival-confirmed

Comment (ryanmansfield) [reply app/views/bookings/_arrival-confirmed.html.erb]: fixed

File: app/views/bookings/_arrival-confirmed.html.erb (hunk 1 above)
Comment (ryanmansfield) [reply app/views/bookings/_arrival-confirmed.html.erb]: fixed

Comment (ryanmansfield) [reply app/views/bookings/_arrival-confirmed.html.erb]: conflicts resolved

**Pull request merged**